src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
```

Comment:
Avoid passing expressions directly into the constructor. It's better to create a variable for the expression result and pass that variable into the constructor. This improves code readability and makes debugging easier if needed.